src/connectWaiter.js

```diff
@@ -68,6 +68,10 @@ export default (params) => {
         }
       }
 
+      componentDidCatch(error) {
+        console.log('redux-waiter: We caught an error in your component. It was caught to prevent the waiter from rejecting your request.')
```

Comment:
Let me know what you think of this. Been looking forward to adding this false "reject" to the promise when the internal component blows up.